cli/commands/login/diff.ts

```diff
@@ -18,7 +18,13 @@ async function diffLoginTemplate() {
   const branding = auth0.branding
   let existingTemplate
   try {
-    existingTemplate = (await branding.getUniversalLoginTemplate()).body
+    const response = await branding.getUniversalLoginTemplate()
+    if (typeof response.data === 'string') {
```

Comment:
I'm confused what this's here fore but seems harmless

Reply:
It's just so typescript doesn't complain. I'm not sure under what circumstances it would return a string but for the template we have the content is under the `body` field